src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/application.ts

```diff
@@ -3,28 +3,48 @@ import { defineStore } from "pinia";
 import { createOrUpdateDraftApplication, getApplications, submitDraftApplication } from "@/api/application";
 import type { ProfessionalDevelopmentExtended } from "@/components/inputs/EceProfessionalDevelopment.vue";
 import type { FileItem } from "@/components/UploadFileItem.vue";
+import applicationWizardAssistantAndOneYear from "@/config/application-wizard-assistant-and-one-year";
+import applicationWizardFiveYear from "@/config/application-wizard-five-year";
+import applicationWizardRenewAssistant from "@/config/application-wizard-renew-assistant";
+import applicationWizardRenewFiveYearActive from "@/config/application-wizard-renew-five-year-active";
+import applicationWizardRenewFiveYearExpiredLessThanFiveYears from "@/config/application-wizard-renew-five-year-expired-less-than-five-years";
+import applicationWizardRenewFiveYearExpiredMoreThanFiveYears from "@/config/application-wizard-renew-five-year-expired-more-than-five-years";
+import applicationWizardRenewOneYearActive from "@/config/application-wizard-renew-one-year-active";
+import applicationWizardRenewOneYearExpired from "@/config/application-wizard-renew-one-year-expired";
 import type { Components } from "@/types/openapi";
-import type { ApplicationStage } from "@/types/wizard";
+import type { ApplicationStage, Wizard } from "@/types/wizard";
 import { humanFileSize } from "@/utils/functions";
 
+import { useCertificationStore } from "./certification";
+import { useUserStore } from "./user";
 import { useWizardStore } from "./wizard";
 export interface ApplicationState {
   applications: Components.Schemas.Application[] | null | undefined;
   draftApplication: Components.Schemas.DraftApplication;
   application: Components.Schemas.Application | null;
 }
 export type ApplicationFlow =
```

Comment:
@SoLetsDev, note change here building exhaustive list of application flows and getting the corresponding configuration file. I figured better do this all in one place...